SparkPost/2web2ui
# FAD-5978 Report: Engagement Data Table

_Refer to [FAD-5978](https://jira.int.messagesystems.com/browse/FAD-5978) for AC._

**Before**

<img width="1386" alt="screen shot 2018-01-23 at 10 18 12 am" src="https://user-images.githubusercontent.com/1335605/35283628-c4da8aa2-0026-11e8-964c-792bf9739379.png">

**After**

<img width="1381" alt="screen shot 2018-01-31 at 10 48 15 am" src="https://user-images.githubusercontent.com/1335605/35632302-4ae80efa-0674-11e8-96cc-2f8b3c837cd5.png">

**How to test**

Confirm the following scenarios work for the whole page (top level metrics, chart, and new table):

- With data (happy path)
- With filtered data (apply any filter ~> table contents change)
- With shared link
- With error

**Remaining Tasks**
- [x] Confirm the table header naming
- [x] Confirm if we need percentage of total clicks column (if so, make performant)
- [x] [Address feedback](https://msys.slack.com/archives/C69L53989/p1517245800000470)

For response to ^^^, [see following comments](https://msys.slack.com/files/U77SVGQP9/F8XEA72UD/screen_shot_2018-01-23_at_12.31.26_pm.png).
